install-nginx.sh

```diff
@@ -1,13 +1,31 @@
 #!/bin/bash
 
```

Comment:
The changes below are part of the effort that @eduardborges in MRADEV-912, MRADEV-911, and MRADEV-913